msbuild/Messaging/Xamarin.Messaging.Build/Xamarin.Messaging.Build.csproj

```diff
@@ -29,18 +28,10 @@
   </ItemGroup>
 
   <ItemGroup>
-      <Reference Include="Microsoft.Build">
-          <HintPath>$(MonoMSBuildBinPath)/Microsoft.Build.dll</HintPath>
-      </Reference>
-      <Reference Include="Microsoft.Build.Framework">
-          <HintPath>$(MonoMSBuildBinPath)/Microsoft.Build.Framework.dll</HintPath>
-      </Reference>
-      <Reference Include="Microsoft.Build.Tasks.Core">
-          <HintPath>$(MonoMSBuildBinPath)/Microsoft.Build.Tasks.Core.dll</HintPath>
-      </Reference>
-      <Reference Include="Microsoft.Build.Utilities.Core">
-          <HintPath>$(MonoMSBuildBinPath)/Microsoft.Build.Utilities.Core.dll</HintPath>
-      </Reference>
+      <PackageReference Include="Microsoft.Build" Version="15.1.548" />
+      <PackageReference Include="Microsoft.Build.Framework" Version="15.1.548" />
+      <PackageReference Include="Microsoft.Build.Tasks.Core" Version="15.1.548" />
+      <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.1.548" />
```

Comment:
@rolfbjarne Any reason to not use latest version? I am just concerned that we may bring security issues this way.

Reply:
I'm not sure if we need to keep the old versions at this point. I think we could try bumping to newer ones and test it. What do the task assemblies depend on?

Reply:
Trying with the latest, let's see what CI says.

@emaf what's the latest desktop TFM that works in VS? Can we use "net4.8.1" or do we have to use something older?

I'm asking because we're building for `netstandard2.0` in some of the projects, and that results in this warning:

> warning NU1701: Package 'Microsoft.Build 17.11.4' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8, .NETFramework,Version=v4.8.1' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.

Reply:
As the build agent does not directly interact with VS, it should not make any difference. I gave this a quick try and seems to work fine.